index.php

```diff
@@ -30,7 +30,7 @@
         break;
 
 // Service Details Pages
-    case preg_match("~^services$~", $url):
+    case preg_match("~^[/]?services$~", $url):
```

Comment:
This small "[/]?" might not seem like it does much, but remember that I said regular expressions were black magic that barely anybody truly understands?  ok, so I will try to explain what this tiny piece of text does.

[] - means character to match, whatever you put inside here, the engine will try to match against the input text

so I put [/] because I want to match the / character, in our situation, we have [/]?services, so I want to match "services" or "/services", you see thats what it is doing, its saying, I want to search for the / character, hence [/] is an expression, you can put multiple characters inside, like [a-z] which means any character between a to z, or [a-zA-Z] which means any a to z character, in lower or upper case

the []? means, whatever characters I search for, I want ZERO OR MORE of them, in other words, zero matches would mean, I want to match it, or not, I just want to see if the / character is there and if it is present, great! match it, if it is not there, still match it, as long as the "services" part of the string is there, it will match

in other words, "[/]?services" means, I want to match "/services" or "services"